src/App/__snapshots__/app.test.js.snap

```diff
@@ -0,0 +1,239 @@
+                type="button"
+                variant="secondary"
+              >
+                Dismiss
+              </Button>
+            }
+            className=""
+            title="Setup Complete"
+            variant="success"
+          >
+            You have successfully launched your patternfly starter project.
+          </Alert>
+        </div>,
+        "className": "app-container",
+      },
+      "ref": null,
+      "rendered": Object {
+        "instance": null,
+        "key": undefined,
+        "nodeType": "host",
+        "props": Object {
+          "children": <Alert
+            action={
+              <Button
+                aria-label={null}
+                className=""
+                component="button"
+                isActive={false}
+                isBlock={false}
+                isDisabled={false}
+                isFocus={false}
+                isHover={false}
+                onClick={[Function]}
+                type="button"
+                variant="secondary"
+              >
+                Dismiss
+              </Button>
+            }
+            className=""
+            title="Setup Complete"
+            variant="success"
+          >
+            You have successfully launched your patternfly starter project.
+          </Alert>,
+          "className": "notification-container",
+        },
+        "ref": null,
+        "rendered": Object {
+          "instance": null,
+          "key": undefined,
+          "nodeType": "function",
+          "props": Object {
+            "action": <Button
+              aria-label={null}
+              className=""
+              component="button"
+              isActive={false}
+              isBlock={false}
+              isDisabled={false}
+              isFocus={false}
+              isHover={false}
+              onClick={[Function]}
+              type="button"
+              variant="secondary"
+            >
+              Dismiss
+            </Button>,
+            "aria-label": undefined,
+            "children": "You have successfully launched your patternfly starter project.",
+            "className": "",
+            "title": "Setup Complete",
+            "variant": "success",
+          },
+          "ref": null,
+          "rendered": "You have successfully launched your patternfly starter project.",
+          "type": [Function],
+        },
+        "type": "div",
+      },
+      "type": "div",
+    },
+  ],
+  Symbol(enzyme.__options__): Object {
+    "adapter": ReactSixteenAdapter {
+      "options": Object {
+        "enableComponentDidUpdateOnSetState": true,
+        "lifecycles": Object {
+          "componentDidUpdate": Object {
+            "onSetState": true,
+          },
+          "getDerivedStateFromProps": true,
+          "getSnapshotBeforeUpdate": true,
+          "setState": Object {
+            "skipsComponentDidUpdateOnNullish": true,
+          },
+        },
+      },
+    },
+  },
+}
+`;
```

Comment:
We could add a snapshot serializer to help make the snapshots more readable.  One i've used is `enzyme-to-json`.  For example, using that module and adding

```
snapshotSerializers: ['enzyme-to-json/serializer']
```
to `jest.config.js`, the above snapshot is reduced to the following

```js
// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests should render default App component 1`] = `
<div
  className="app-container"
>
  <div
    className="notification-container"
  >
    <Alert
      action={
        <Button
          aria-label={null}
          className=""
          component="button"
          isActive={false}
          isBlock={false}
          isDisabled={false}
          isFocus={false}
          isHover={false}
          onClick={[Function]}
          type="button"
          variant="secondary"
        >
          Dismiss
        </Button>
      }
      className=""
      title="Setup Complete"
      variant="success"
    >
      You have successfully launched your patternfly starter project.
    </Alert>
  </div>
</div>
`;
```

Reply:
This is awesome! Definitely adding this.